gpiozero/boards.py

```diff
@@ -265,7 +265,7 @@ class LEDBarGraph(LEDCollection):
     """
 
     def __init__(self, *pins, **kwargs):
-        super(LEDBarGraph, self).__init__(*pins, pwm=False)
+        super(LEDBarGraph, self).__init__(*pins, **kwargs)
         initial_value = kwargs.get('initial_value', 0)
```

Comment:
Sorry, I meant `*pins, initial_value=0` - I was thinking of LEDBoard (which has `pwm=False`, but which I see I need to add `active_high` and `initial_value` to).

On the caveat of keyword only parameters - I'm not sure we have to explain that. Any parameters after an *args style parameter are implicitly keyword only (as [PEP-3102](https://www.python.org/dev/peps/pep-3102/), which introduced keyword only parameters, specifies) because if they weren't keywords the *args would suck them up. But then given this is Python 3 only functionality, I could well believe there's plenty of Python 2 users out there that this wouldn't be immediately obvious to. Could be worth adding something like a "(keyword-only)" tag to such parameters in the docs...